Common/Utils.hs

```diff
@@ -184,6 +185,13 @@ readMaybe s = case filter (all isSpace . snd) $ reads s of
   [(a, _)] -> Just a
   _ -> Nothing
 
+-- | Left-biased choice on maybes
+-- Extracted from Data.Generics.Aliases (syb-0.4.4)
+orElse :: Maybe a -> Maybe a -> Maybe a
+x `orElse` y = case x of
```

Comment:
I'm no friend of using alphanumeric identifiers as infix operators. 
A shorter implementation would be:
`orElse x y = maybe y Just x`